Library/Formula/apcupsd.rb

```diff
@@ -0,0 +1,106 @@
+class Apcupsd < Formula
+  desc "Daemon for controlling APC UPSes"
+  homepage "http://www.apcupsd.org"
+  url "https://downloads.sourceforge.net/project/apcupsd/apcupsd%20-%20Stable/3.14.13/apcupsd-3.14.13.tar.gz"
+  sha256 "57ecbde01d0448bf8c4dbfe0ad016724ae66ab98adf2de955bf2be553c5d03f9"
+
+  option "with-usb", "Build with support for USB UPSes."
+
+  depends_on "gd"
+  depends_on "libusb-compat"
+
+  def install
+    cd "src/apcagent" do
+      # Install apcagent.app to `prefix` instead of /Applications.
+      inreplace "Makefile", "Applications", "#{"#{prefix}".gsub(%r{/^\//}, "")}"
```

Comment:
The upstream developer told me this needs to be hardcoded for the creation of the `.pkg` installer, at least for now. The same goes for the lines below as these are also part of the `.pkg`.